hokusai/development.yml

```diff
@@ -14,7 +14,7 @@ services:
     env_file:
       - ../.env.shared
       - ../.env
-    command: ["yarn", "start:dev"]
+    command: ["yarn", "start"]
```

Comment:
While testing this ended up running Force via `hokusai dev start` and turns out the command is out of date.